frontend/src/components/TimeTravel.tsx

```diff
@@ -28,6 +38,17 @@ export const TimeTravel = ({
     setCurrentWeek(weekNumber(nextDate));
   };
 
+  const CustomDatePickerInput = forwardRef(({ value, onClick }, ref) => (
+    <label onClick={onClick} className="header-time-travel" ref={ref}>
+      Week {`${currentWeek}`}
+    </label>
```

Comment:
`<label>` is supposed to be used together with an `<input>` tag, so it shouldn't be used here. Best from a semantic point of view would be to use a `<button>` tag (it doesn't have to look like a button).  
It's important so that screen readers recognize it as something clickable.